src/ObjCommon/Game/T6/ObjConstantsT6.h

```diff
@@ -24,5 +26,14 @@ namespace T6
         static constexpr const char* GDF_FILENAME_WEAPON_ATTACHMENT = "attachment.gdf";
         static constexpr const char* GDF_FILENAME_WEAPON_ATTACHMENT_UNIQUE = "attachmentunique.gdf";
         static constexpr const char* GDF_FILENAME_ZBARRIER = "zbarrier.gdf";
+
+        static const std::array<std::string, 27> SOUND_GROUPS;
```

Comment:
The reason I wrote it this way is from a code readability standpoint, I felt it was a lot easier to read `SOUND_GROUPS.size()` rather than `std::extent_v<decltype(SOUND_GROUPS)>`. When building the sound aliases I have to reference these lists and their size a lot. I didn't want to use a vector since its all constants, but this felt like an okay in between. I can swap it back if you'd prefer